src/Commands/Self/Plugin/InstallCommand.php

```diff
@@ -42,6 +43,38 @@ public function install(array $projects)
         }
     }
 
+    /**
+     * Migrate Terminus 2 plugins.
+     *
+     * @command self:plugin:migrate
+     * @aliases plugin:migrate
+     *
+     * @throws \Pantheon\Terminus\Exceptions\TerminusException
+     */
+    public function migrate()
+    {
+        $default_plugins_dir = getenv('HOME') . DIRECTORY_SEPARATOR . '.terminus' . DIRECTORY_SEPARATOR . 'plugins';
+        $plugins_dir = !empty(getenv('TERMINUS2_PLUGINS_DIR')) ? getenv('TERMINUS2_PLUGINS_DIR') : $default_plugins_dir;
```

Comment:
This should probably use config instead of directly an env var: `$this->getConfig()->get('plugins_dir_terminus2');`

Then, it could be added to config/constants as TERMINUS_PLUGINS_DIR_TERMINUS2 and you could also override it with an env var TERMINUS_PLUGINS_DIR_TERMINUS2

Reply:
That makes sense.